slides/bbs-mmrm/slides.qmd

```diff
@@ -0,0 +1,127 @@
+---
+title: "mmrm"
+author: "Daniel Sabanes Bove"
+institute: (Statistical Engineering,
+  Pharma Product Development Data Sciences, 
+  Roche)
+date: "@BBS/EFSPI workshop 8 Dec 2022"
+format: 
+  revealjs:
+    incremental: true
+    logo: sticker-1200.png
+    slide-number: c/t
+title-slide-attributes: 
+  data-background-image: mmrm.png
+  data-background-size: 20%
+  data-background-opacity: "0.5"
+  data-background-position: 80% 10%
+---
+
+## Acknowledgments
+
+::: nonincremental
+::: columns
+::: {.column width="50%"}
+Authors:
+
+-   Kevin Kunzmann (Boehringer Ingelheim)
+-   Brian Matthew Lang (MSD)
+-   Ya Wang (Gilead)
+-   Julia Dedic (Roche)
+-   Doug Kelkhoff (Roche)
+-   Liming Li (Roche)
+:::
+
+::: {.column width="50%"}
+Thanks to:
+
+-   Ben Bolker (McMaster University)
+-   Software Engineering working group (SWE WG)
+-   Gonzalo Duran Pacheco (Roche)
+-   Craig Gower-Page (Roche)
+-   Dinakar Kulkarni (Roche)
+-   Davide Garolini (Roche)
+:::
+:::
+:::
+
+## Motivation
+
+-   2 years ago, we thought we had solved the MMRM problem, using `lme4` and `lmerTest`
+-   However in March this year, Gonzalo and Julia told us the bad news: this approach failed on their Ophthalmology data sets (slow, did not converge)
+-   `nlme` does not give Satterthwaite adjusted degrees of freedom, and with `emmeans` it is only approximate, also has convergence issues
+-   Next we tried to extend `glmmTMB` to calculate Satterthwaite adjusted degrees of freedom. But it did not work.
+
+## Idea
+
+-   Because `glmmTMB` is always using a random effects representation, cannot have a real unstructured model (need to use a trick)
```

Comment:
```suggestion
-   Because `glmmTMB` is always using a random effects representation, we cannot have a real unstructured model (need to use a trick)
```